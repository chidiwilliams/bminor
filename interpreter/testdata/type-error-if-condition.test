x: integer = 65;
y: char = 'A';
if (x > y) {
    print x;
}
