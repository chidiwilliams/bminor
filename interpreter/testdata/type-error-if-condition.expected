err:Error on line 3: 'x' and 'y' are of different types
err: